src/Display/Modal/Modal.test.tsx

```diff
@@ -96,6 +96,16 @@ describe('when modal is closed', () => {
   });
 });
 
+it('children should not be mounted', () => {
```

Comment:
should we change this to `children should not be mounted when modal isn't visible`? :D

Reply:
Oh I intended to put inside `describe('when modal is closed', () => {})` haha. Just changed it, thanks!